best_practices/controllers.rst

```diff
@@ -47,14 +47,14 @@ this suffix is neither required nor recommended, so you can safely remove it.
 Routing Configuration
 ---------------------
 
-To load routes defined as annotations in your controllers, add the following
-configuration to the main routing configuration file:
+To load routes defined as annotations in your controllers and if you are not using Symfony Flex, add the following
+configuration to the annotation routing configuration file:
```

Comment:
Great find with this. I think we can go further and fully assume Flex and reword this section to read something more like this:

> To load routes defined as annotations in your controllers, run “composer require doctrine/annotations”. Thanks to the Flex recipe, a config/routes/annotations.yaml file will be created:

... show the file

... then continue on like normal with “This Configuration...”

Reply:
Should we spread this modification to other branches ?
If so, i guess i should duplicate the PR once you validate it ? Or will you be able to do it automatically ?